docs/guides/edgemesh_test_env_guide.md

```diff
@@ -12,12 +12,40 @@ edged:
 - install CNI plugin
 
 # Edgemesh test env config guide
+## Docker support (probation)
```

Comment:
It's best to have an overview like this: we support docker and containerd, recommend docker